data/remnant missions.txt

```diff
@@ -495,3 +560,718 @@ mission "Remnant: Technology Available"
+
+
+
+mission "Remnant: Return the Samples"
+	name "Return the Samples"
+	description "A researcher on <planet>, has asked you to return some samples to Nasqueron."
+	source Aventine
+	stopover Nasqueron
+	to offer
+		has "Remnant: Technology Available: offered"
+		random < 50
+	on offer
+		conversation
+			`As you are walking through the spaceport, the researcher you previously assisted with the studies on the void sprites approaches you in a hustle. "We have made some new discoveries!" he exclaims in a trill. "Some of the samples that we took all those years ago are actually eggs of some kind. So the Archon must view our previous research as being disruptive to the void sprites' lifecycles.`
+			`	"We have been storing these eggs in a cryogenic stasis tank, so they should still be viable and intact. Well, most of them, anyway." His notes trail off on a mournful tone.`
+			`	"Could you return some of them to Nasqueron? According to our logs, they were found floating in the lower levels of the atmosphere, where their natural buoyancy should keep them afloat."`
+			choice
+				`	"Sure, I would be glad to return them."`
+				`	"Sorry, I'm really not interested in braving the Archon again."`
+					decline
+
+			`	"Thank you. Since we are continuing to work together, I'll try again." He takes a breath, concentrates, and speaks a short verse about plumage with a complex chord that sounds at once grandiose and self-deprecating. The best you can make out is "Plume." "We can't pay you for this, but if it does appease the Archon we will be able to resume our studies, and who knows where that could lead?"`
+				accept
+	npc
+		government " Drak "
+		system "Nenia"
+		personality staying heroic nemesis uninterested
+		ship "Archon (Cloaked)" "Lifted Lorax"
+	on accept
+		log "People" "Plume" `A Remnant researcher specializing in xenobiology, Plume is at the forefront of recent efforts to restart research on the void sprites and a leading expert in non-carbon based lifeforms.`
+	on stopover
+		conversation
+			`Once again you have successfully dodged the Archon to land on Nasqueron. Somehow, knowing it was going to be there was both better and worse than the surprise of the first time.`
+			`	As <ship> sinks through the cloud layers, you gently activate the repulsors to slow your descent to avoid startling the void sprites. After what seems like an eternity, you notice a few nearby sprites descending as well, and you match velocity with them, slowing as you reach the altitude listed in the log.`
+			`	Setting the ship on auto-pilot, you head back and suit-up in something that looks more reminiscent of the old deep-sea diving suits than anything carried on a modern ship. Readying the eggs in the cargo hold, you equalize pressure in the main cargo hold to the swirling clouds outside, and open the bay doors. As predicted, at this pressure level, the eggs are almost weightless, and are fairly easy to carry to the door of the ship and release.`	
```

Comment:
We do have dump cargo buttons. I'd shorten this down.
This entire paragraph -> `You hit the button to dump the eggs from your cargo hold. As predicted at this pressure level, the eggs are almost weightless as they float out of your cargo hold and into the atmosphere of the gas giant.`

Reply:
Hmmm, you've never done wildlife re-introduction things, have you? Hitting the "dump cargo" button is for ditching garbage and/or stuff you hope will distract a pirate, and in either case is stuff that one isn't really caring about any more. It is *not* a proper and careful way to put eggs back into a natural environment. Not really conducive to a healthy release.

That being said, having the player release them carefully is a better explanation for the Archon's change in behaviour. 

Likewise, I was specifically intending this to have a more personal touch.

Reply:
There is something quite special about doing it this way, sure it's not the most practical or likely but it is a really nice way of doing it.

Reply:
Thanks. 

As far as that goes, it also wouldn't work: The description specifies that the eggs are stored in cocoons (aka packages to keep them safe in transit), and the player is removing them from the storage to release them out of the ship. Chances are the weight of the packaging might be enough to cause them to plummet through the atmosphere, and they'd probably be unrecognizable to the Void Sprites too. Be along the same lines as giving an egg carton full of eggs to a chicken and hoping it has the smarts to open it and incubate them. Maybe it surprises you, but probably you just end up with dead eggs. Actually take the eggs out, on the other hand, and there's decent chances of it being taken care of. (varies widely with the species of creature being dealt with, but as a general rule, anyway)

Reply:
Fin that’s the case, then Plume should explicitly state that you need to release the eggs in this way. Perhaps he should say something about how delicate they are and that you should personally release them instead of simply purging your cargo bay of them.

Reply:
OK, that should be fairly straight forward to do.

Reply:
Just reading over my rewritten version with the description of the suit in the first part instead of the second part, and I don't think that it really provides any benefit moving that description over.